test/unit/test-amp-img.js

```diff
@@ -837,5 +839,116 @@ describes.sandboxed('amp-img', {}, (env) => {
             });
           });
       });
+
+      it('SSR sizer does not interfere with img creation', () => {
+        let ampImg;
+        const parentDiv = fixture.doc.getElementById('parent');
+        parentDiv.setAttribute('style', 'width: 80px; height: 80px');
+
+        // Hack so we don't duplicate intrinsic's layout code here.
+        const tmp = createElementWithAttributes(fixture.doc, 'div', {
+          src: '/examples/img/sample.jpg', // 641 x 481
+          width: 800,
+          height: 600,
+          layout: 'intrinsic',
+        });
+        applyStaticLayout(tmp);
+        const attributes = {
+          'i-amphtml-ssr': '',
+        };
+        for (let i = 0; i < tmp.attributes.length; i++) {
+          attributes[tmp.attributes[i].name] = tmp.attributes[i].value;
+        }
```

Comment:
Nit: Can we dedupe this in a nested `describe('SSR sizer does not interfere with img')` block and `beforeEach`?